### PR TITLE
overview

### PR DESCRIPTION
**Pull request by wwarner-salesforce**
ACM docs: Overview draft.

### CODE REVIEW

File: modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -4,7 +4,13 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
-API Community Manager (ACM) overview here.
+MuleSoft enables your company to build APIs, catalog them with Exchange, and present them to software developers inside and outside your company with API Community Manager (ACM).
```
Comment (valkyrie69) [inline modules/ROOT/pages/index.adoc]: suggestion only (take or leave)

API Community Manager (ACM) enables your company to build APIs, catalog them with Anypoint Exchange, and make them available to software developers inside and outside of your company.

File: modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -4,7 +4,13 @@ include::_attributes.adoc[]
...
+
+ACM relies on MuleSoft's Anypoint Exchange as the single source of truth about published APIs, and Salesforce Community Cloud's ability to add customization, branding, and marketing information and to serve different content to different audiences.
```
Comment (valkyrie69) [inline modules/ROOT/pages/index.adoc]: ...Salesforce Community Cloud's ability to add customization, branding, and marketing information to serve unique content to different audiences.

(suggestion only)

File: modules/ROOT/pages/index.adoc [hunk 3]
```diff
@@ -4,7 +4,13 @@ include::_attributes.adoc[]
...
+
+With ACM your company can provide its APIs to internal and external developers, treat the APIs as a product and the developers as customers, and provide rich features such as chat, forums, and support integration.
```
Comment (valkyrie69) [inline modules/ROOT/pages/index.adoc]: ACM provides an array of features, such as chat, forums, and support integration. This enables your company to treat its APIs as product, and treat developers as customers.  

(suggestion)

File: modules/ROOT/pages/index.adoc [hunk 4]
```diff
@@ -4,7 +4,13 @@ include::_attributes.adoc[]
...
+
+ACM also introduces API Products. An API Product is a group of APIs with marketing context. For example, a payments product might include three APIs for Apple pay, Android pay, and wire transfers.
```
Comment (valkyrie69) [inline modules/ROOT/pages/index.adoc]: I don't think product should be capped (unless it is a product name)

**Review by valkyrie69 — APPROVED** [review]
I left some suggestions for wording, which you can take or leave.

File: modules/ROOT/pages/index.adoc (hunk 1 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/index.adoc]: Thank you. I like that better.

File: modules/ROOT/pages/index.adoc (hunk 2 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/index.adoc]: Updated to that version.

File: modules/ROOT/pages/index.adoc (hunk 3 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/index.adoc]: Much better! Updated.

File: modules/ROOT/pages/index.adoc (hunk 4 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/index.adoc]: Yes, now that you mention it, I think that should be lowercase. I'm not certain so I'll use lowercase for now and make a note to ask.

**Comment by wwarner-salesforce** [comment]
Suggestions are always helpful, and these were top notch. Thank you!

**Pull request merged**